app/models/job.rb

```diff
@@ -178,6 +178,11 @@ def self.check_jobs_for_timeout
   def timeout_adjustment
     timeout_adjustment = 1
     target = target_entity
+    ems_settings = target.try(:ext_management_system).try(:custom_attributes)
```

Comment:
@enoodle I think we should stop using the `custom_attributes` to extend settings for now until we'll have something more consistent.